@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1665,15 +1687,14 @@ export class TextElementsRenderer {
                 }
 
                 // Allocate collision info if needed.
```

Comment:
Not allocating screen space for fading out labels causes a flickering effect when a label fades out while the same labels at another zoom level fades in in the same place, visible in berlin style where we have a font background color.

Reply:
thanks, that makes sense